src/Sylius/Bundle/PayumBundle/DependencyInjection/SyliusPayumExtension.php

```diff
@@ -44,4 +45,23 @@ public function load(array $config, ContainerBuilder $container)
         $container->setParameter('payum.template.layout', $config['template']['layout']);
         $container->setParameter('payum.template.obtain_credit_card', $config['template']['obtain_credit_card']);
     }
+
+
+    /**
+     * {@inheritdoc}
+     */
+    public function prepend(ContainerBuilder $container)
+    {
+        if (!$container->hasExtension('sylius_payment')) {
+            return;
+        }
+
+        $config = $container->getExtensionConfig('payum');
```

Comment:
Sure they can. However this is just a prepend, gateways configuration can be easily overwritten in ``yml`` file, or in extension, or in any other possible way.